### DISCUSSION TITLE
Product attribute group and attribute not appearing

### DISCUSSION THREAD

**Discussion by SebastianBotez** [lunarphp/lunar#1495] [Q&A] [answered]
Hello guys.
Just discovered yesterday this awesome project.
I have a problem adding Attribute to products. Simply it does not appear in the product page (same with product variant).
For example:

<img width="1662" alt="Screenshot 2024-01-24 at 10 52 29" src="https://github.com/lunarphp/lunar/assets/10911720/e34b674d-3306-4896-aaf4-1f6a8de5041e">

I have set the product SEO data but it does not appear in product.
Is there any extra config setting that I'm missing?

I'm using:
Lunar: 0.7.5
Laravel: 10.41

Appreciate your help guys.

**Comment by glennjacobs** [comment, marked as answer]
Have you mapped the attributes to the product type?

**Reply by SebastianBotez** [reply]
Well ... believe me or not, I searched for almost 3 hours before asking. Dummy me.
Thank you for help! Great product! Can't wait to get the shop done!
